docs/api/apiv3/paths/storage_files_prepare_upload.yml

```diff
@@ -64,8 +64,17 @@ post:
             errorIdentifier: urn:openproject-org:api:v3:errors:NotFound
             message: The requested resource could not be found.
       description: |-
-        Returned in either of those cases:
-        - if the storage does not exist or the client does not have sufficient permissions to see it
+        Returned the storage does not exist or the client does not have sufficient permissions to see it.
```

Comment:
That sentence is somehow looking weird. Please have a look again. It doesn't tell me...

Reply:
Maybe an 'if' is missing.

Reply:
added `if`